packages/storybook/src/stories/grid-fluid.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import { XGrid } from '@material-ui/x-grid';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import '../style/grid-stories.css';
+
+export default {
+  title: 'X-Grid Tests/Fluid Column Width',
+  component: XGrid,
+  decorators: [withKnobs, withA11y],
+  parameters: {
+    options: { selectedPanel: 'storybook/storysource/panel' },
+    docs: {
+      page: null,
+    },
+  },
+};
+export const FewColumns = () => {
+  const rows = [
+    {
+      id: 1,
+      '2M': '2',
+      '3M': '3',
+    },
+  ];
+  const columns = [
+    {
+      field: 'id',
+      flex: 3,
+    },
+    {
+      field: '2M',
+      width: 200,
+    },
+    {
+      field: '3M',
+      flex: 1,
+    },
+  ];
+
+  return (
+    <div className="grid-container">
+      <XGrid rows={rows} columns={columns} />
+    </div>
+  );
+};
+
+export const SeveralColumn = () => {
+  const rows = [
```

Comment:
why not use useDemoData with a transform function

Reply:
```
export function SmallColSizes() {
  const data = useData(100, 20);
  const transformColSizes = (columns: ColDef[]) => columns.map((c) => ({ ...c, width: 60 }));

  return <XGrid rows={data.rows} columns={transformColSizes(data.columns)} />;
}
```

Reply:
done